README.md

```diff
@@ -1,6 +1,9 @@
 # workflow.portfolio.parsing
-
-[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
+<!-- badges: start -->
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
```

Comment:
Do `lifecycle` and `repo-status` serve redundant purposes here? Again, feels like a practice we need to collectively define and understand as a team before using too many badges, otherwise we risk having inconsistent interpretations. 

Relates strongly to: https://github.com/RMI-PACTA/practices/issues/6

Reply:
Hmm. Possibly, but I see them as different? I'll add thoughts in RMI-PACTA/practices#6.

Reply:
Thank you! 

I am genuinely asking, I don't know/ understand the difference. Just hoping to ensure we get the whole team on the same page.